samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -782,7 +782,9 @@ int main(void)
 
 		printk("Advertising successfully started\n");
 
-		k_work_init_delayable(&audio_send_work, audio_timer_timeout);
+		if (CONFIG_BT_ASCS_ASE_SRC_COUNT) {
+			k_work_init_delayable(&audio_send_work, audio_timer_timeout);
+		}
```

Comment:
Ditto: Not related to the overlay files. 

```suggestion
		if (IS_ENABLED(CONFIG_BT_ASCS_ASE_SRC_COUNT)) {
			k_work_init_delayable(&audio_send_work, audio_timer_timeout);
		}
```